backend/LexBoxApi/Services/HgService.cs

```diff
@@ -267,6 +267,22 @@ private string[] GetWsList(System.Xml.XmlElement root, string tagName)
         return null;
     }
 
+    public async Task<int?> GetModelVersionOfFlexProject(ProjectCode code, CancellationToken token = default)
+    {
+        var result = await ExecuteHgCommandServerCommand(code, "flexmodelversion", token);
+        var text = await result.ReadAsStringAsync(token);
+        try
+        {
+            var json = JsonDocument.Parse(text);
+            return json.RootElement.GetProperty("modelversion").GetInt32();
+        }
+        catch
+        {
+            if (int.TryParse(text, out var num)) return num;
```

Comment:
Fallback in case a future FLEx or Chorus update decides to make this a simple text file containing the number instead of JSON. Probably not needed; can remove if you think I should remove it.

Reply:
Done.